README.md

```diff
@@ -36,16 +36,22 @@ model.load_model('model.bin')
 emb = model.embed_sentence("once upon a time .") 
 embs = model.embed_sentences(["first sentence .", "another sentence"])  
 ```
-In order to compile and install the module globally, run the following from the `src` folder:
+In order to compile and install the module, run the following from the project root folder:
 
 ```
-python setup.py build_ext
-sudo pip install .
+pip install .
 ```
 Text preprocessing (tokenization and lowercasing) is not handled by the module, check `wikiTokenize.py` for tokenization using NLTK and Stanford NLP. 
 
 An alternative to the Cython module is using the python code provided in the `get_sentence_embeddings_from_pre-trained_models` notebook. It handles tokenization and can be given raw sentences, but does not keep the model in memory. 
 
+#### Running inference with multiple processes
+
+There is an 'inference' mode for loading the model in the Cython module, which loads the model's input matrix into a shared memory segment and doesn't load the output matrix, which is not needed for inference. This is an optimization for the usecase of running inference with multiple independent processes, which would otherwise each need to load a copy of the model into their address space. To use it:
+```python
+model.load_model('model.bin', inference_mode=True)
+```
+
```

Comment:
Thanks for the changes, one last thing, can we just have a small comment on how to free the memory in the README?

Reply:
Updated